src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -844,9 +915,14 @@ def add_plane_segmentation(
             )
 
         image_segmentation.add_plane_segmentation(plane_segmentations=[plane_segmentation])
+        image_segmentation.add_plane_segmentation(plane_segmentations=[background_plane_segmentation])
     return nwbfile
 
 
+def add_background_plane_segmentation():
```

Comment:
Exactly 😉 I would prefer this. Then perhaps an added `add_all_plane_segmentations` that calls each individual single plane segmentation addition, unless we think that's too much redirection